### ISSUE TITLE
System.AggregateException right after startup

### ISSUE THREAD

**Issue by homoroselaps** [Wox-launcher/Wox#1480]
Wox version: 1.3.424
OS Version: Microsoft Windows NT 10.0.14393.0
IntPtr Length: 8
x64: True

Python Path: 
Everything SDK Path: C:\Users\jt-1\AppData\Roaming\Wox\Settings\Plugins\Wox.Plugin.Everything\EverythingSDK\x64\Everything.dll
Date: 06/18/2017 19:33:31
Exception:
mscorlib
System.AggregateException
One or more errors occurred.
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Parallel.ForWorker[TLocal](Int32 fromInclusive, Int32 toExclusive, ParallelOptions parallelOptions, Action1 body, Action2 bodyWithState, Func4 bodyWithLocal, Func1 localInit, Action1 localFinally)
   at System.Threading.Tasks.Parallel.ForEachWorker[TSource,TLocal](IEnumerable1 source, ParallelOptions parallelOptions, Action1 body, Action2 bodyWithState, Action3 bodyWithStateAndIndex, Func4 bodyWithStateAndLocal, Func5 bodyWithEverything, Func1 localInit, Action1 localFinally)
   at System.Threading.Tasks.Parallel.ForEach[TSource](IEnumerable1 source, Action1 body)
   at Wox.Core.Plugin.PluginManager.InitializePlugins(IPublicAPI api) in C:\projects\wox\Wox.Core\Plugin\PluginManager.cs:line 91
   at Wox.App.<OnStartup>b__10_0() in C:\projects\wox\Wox\App.xaml.cs:line 60
   at Wox.Infrastructure.Stopwatch.Normal(String message, Action action) in C:\projects\wox\Wox.Infrastructure\Stopwatch.cs:line 31
   at System.Windows.Application.OnStartup(StartupEventArgs e)
   at System.Windows.Application.<.ctor>b__1_0(Object unused)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)

**Comment by lz42** [comment]
Same issue.

[2017-06-25.txt](https://github.com/Wox-launcher/Wox/files/1100217/2017-06-25.txt)

Wox version: 1.3.424
OS Version: Microsoft Windows NT 10.0.14393.0
IntPtr Length: 8
x64: True

**Comment by lances101** [comment]
It seems like you are using "Wox.Plugin.Simpleclock", which is not an official Wox plugin therefore we cannot provide support for it. The plugin is not compatible with the latest version of Wox and therefore is crashing it. 

You can ask the developer to update it, or you can fork, update and upload it yourself as a new plugin.

In order to continue using Wox you need to delete this plugin. This can be done by exiting Wox, going to `%appdata%/Wox/Plugins` and deleting the corresponding plugin's folder.

Closed due to being a third-party plugin.

**Comment by lz42** [comment]
I want to add just a few words. I've resolved problem by myself and simpleClock works flawlessly with Wox 1.3.183. So the story goes like that: I like to be always up to date, so I've installed every new release of Wox **over the old version**. But suddenly wox became very unstable, it crashed constantly (this was one of the issues). Reinstalation didn't help much. It made me so angry that I've switched to another launcher. But then I got an idea: I've uninstalled wox, and cleaned (**removed**) all remaining folders on hdd especially **plugins**. After clean install of wox and reinstalation of all plugins everything works flawlessly to this day. Happy ending. Cheers.
